proposals/simd/SIMD.md

```diff
@@ -188,15 +208,17 @@ The `_s` and `_u` variants will sign-extend or zero-extend the lane value to
 `i32` respectively.
 
 ### Replace lane value
-* `i8x16.replace_lane(a: v128, i: LaneIdx16, x: i32) -> v128`
-* `i16x8.replace_lane(a: v128, i: LaneIdx8, x: i32) -> v128`
-* `i32x4.replace_lane(a: v128, i: LaneIdx4, x: i32) -> v128`
-* `i64x2.replace_lane(a: v128, i: LaneIdx2, x: i64) -> v128`
-* `f32x4.replace_lane(a: v128, i: LaneIdx4, x: f32) -> v128`
-* `f64x2.replace_lane(a: v128, i: LaneIdx2, x: f64) -> v128`
-
-Return a new vector with lanes identical to `a`, except for lane `i` which has
-the value `x`.
+* `i8x16.replace_lane(a: v128, imm: ImmLaneIdx16, x: i32) -> v128`
+* `i16x8.replace_lane(a: v128, imm: ImmLaneIdx8, x: i32) -> v128`
+* `i32x4.replace_lane(a: v128, imm: ImmLaneIdx4, x: i32) -> v128`
+* `i64x2.replace_lane(a: v128, imm: ImmLaneIdx2, x: i64) -> v128`
+* `f32x4.replace_lane(a: v128, imm: ImmLaneIdx4, x: f32) -> v128`
+* `f64x2.replace_lane(a: v128, imm: ImmLaneIdx2, x: f64) -> v128`
+
+Return a new vector with lanes identical to `a`, except for the lane specified
+in the immediate mode operand `imm` which has the value `x`. The
+`{interpretation}.replace_lane` instructions are encoded with one immediate byte
+providing the index of the lane to whose value is to be replaced.
```

Comment:
Language nit: Would it be okay to change the last bit of the sentence to  "instructions are encoded with an immediate byte providing the index of the lane the value of which is to be replaced."?

Reply:
Sure, thank you!